sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -244,6 +244,7 @@ public boolean isInternal() {
 
   class NewAction {
     private final String key;
+    private final String PAGE_PARAM_DESCRIPTION = "1-based page number";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "PAGE_PARAM_DESCRIPTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S00116](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00116))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWGfL87_R7xclXvFfE0r&open=AWGfL87_R7xclXvFfE0r)

[ISSUE_KEY]: AWGfL87_R7xclXvFfE0r